README.md

```diff
@@ -2,3 +2,19 @@
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9272daf4b714e4f95280916e763b6b2)](https://www.codacy.com/app/HMCTS/bulk-scan-orchestrator)
+
+## Running end to end using docker
+- Run up the docker environment from bulk-scan-shared-infrastructure
+- You will have to setup these environment variables on either your
+run configuration or bash shell
+```$bash
```

Comment:
isn't the syntax ```bash i.e. without the $?

Reply:
hmm ... intellij seems to think it was with the $

Reply:
https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown